src/components/Player.js

```diff
@@ -147,6 +148,18 @@ export default class Player extends Component {
       <CueBar key="cue-bar" order={6.0} />,
```

Comment:
I could have just removed this, but figured this is a better approach since one might not always want to create unique cuebar?